### PR TITLE
hw-5.3

### PR DESCRIPTION
**Pull request by julia-shvabovich**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -8,7 +14,26 @@
...
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder fileContent = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            while (reader.ready()) {
+                fileContent.append(reader.readLine());
+            }
+        } catch (Exception e) {
+            System.out.println("The file with such name does not exist");
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/FileWork.java:26]: https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_05.html
Don’t ignore exceptions. Re-throw them as RuntimeException(e.g. throw new RuntimeException(e)) if you don’t expect your code to fail here, or make necessary actions at the catch block to fix the problem.

Comment (jlightgreen) [reply src/main/java/core/basesyntax/FileWork.java]: done

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -8,7 +14,26 @@
...
  */
 public class FileWork {
+    private static final String FIRST_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder fileContent = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            while (reader.ready()) {
+                fileContent.append(reader.readLine());
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:23]: it seems the lines are not separate from each other after you append them to StringBuilder

Comment (jlightgreen) [reply src/main/java/core/basesyntax/FileWork.java]: done

**Pull request closed without merging**